duckduckgo/macos-browser
# Remove lock screen

Task/Issue URL: https://app.asana.com/0/1199230911884351/1203111586182492/f
Tech Design URL: 
CC:

**Description**:

This PR removes the lock screen and related waitlist code.

It also fixes a bug where the `onboading.complete` user defaults value was not set if you chose to set the browser as a default (even if you canceled the default browser prompt after it was presented).



**Steps to test this PR**:
1. Run the browser and check that you don't get the onboarding step in debug mode
1. Run the production version of the browser off of this branch (the Pull Request Testing scheme should do it) and check that you don't get onboarding, as you have presumably already been through it
1. Run `./clean-app.sh review` to remove your product review build install data. Then, run the Product Review scheme and verify that you do get the onboarding. Skip all of the onboarding steps and then reach the end of the flow - after that, run the app again and check that onboarding does not appear.

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
